server/src/pages/shot/view.js

```diff
@@ -528,6 +515,18 @@ class Body extends React.Component {
     // Note: we allow the default action to continue
   }
 
+  onClickFavorite() {
+    if (this.props.expireTime) {
+      sendEvent("set-favorite", "navbar");
+      const INDEFINITE = 0;
+      this.props.controller.changeShotExpiration(this.props.shot, INDEFINITE);
+    } else {
+      sendEvent("unset-favorite", "navbar");
+      const TWO_WEEKS_IN_SECONDS = 1209600;
+      this.props.controller.changeShotExpiration(this.props.shot, Date.now() + TWO_WEEKS_IN_SECONDS);
```

Comment:
Date.now() here will be in milliseconds and should be updated to use Date.now() /1000

Reply:
Good catch.  The value should be in ms not seconds.

Reply:
cool, update metrics.md should be updated with new event 'unset-favorite' and 'set-favorite'

Reply:
and remove ones which are no longer used such as set-expiration-to-indefinite